src/effects/index.ts

```diff
@@ -75,6 +75,20 @@ type FireworksOptions = {
      */
     gravity: number;
 }
+type SnowfallOptions = {
+    /**
+     * The maximum number of snowflakes to render at a given time
+     */
+    maxCount: number;
+    /**
+     * The amount of gravity to apply to the snowflakes
+     */
+    gravity: number;
+    /**
+     * The amount of drift (horizontal sway) to apply to the snowflakes. Each snowflake varies.
+     */
+    maxDrift: number;
+}
```

Comment:
ftr this sort of duplication is a function of how the effects are structured. Ideally they are DRY, but that would (currently) cause import loops. Easy improvements can be made in a future PR before the fourth effect.